minijanus.js

```
@@ -89,8 +89,19 @@ JanusSession.prototype.create = function() {
 /** Destroys this session. **/
 JanusSession.prototype.destroy = function() {
   // Note that this message, unlike others, does *not* receive a response from janus, so there is no promise returned.
-  this.send("destroy");
+  try {
+    this.send("destroy").catch((e) => {});
```

Why the empty catch handler here?